doc/widgets/networkofgroups.md

```diff
@@ -0,0 +1,37 @@
+Network of Groups
+=================
+
+Group instances by feature and connect related groups.
+
+**Inputs**
+
+- Network: An instance of network graph.
+- Data: Properties of a network graph.
+
+**Outputs**:
+
+- Network: A grouped network graph.
+- Data: Properties of the group network graph.
+
+**Network of Groups** is the network version of the group-by operation. Nodes with the same values of the attribute, selected in the dropdown, will be represented as a single node.
+
+![](images/Network-of-Groups-stamped.png)
+
+1. Information on the input and output network.
+2. Select the attribute to group by.
+3. Compute weights:
+   - *No weights*: all weights are set to 1.
+   - *Number of connections*: edge weight is the sum of connections in common.
+   - *Sum of connection weights*: edge weight is the sum of weights of connections in common.
+   *Normalize by geometric mean* divides weights by the square root of the product of degrees of edge pairs.
```

Comment:
@janezd Please check this.

Reply:
What about this?

"""
- *Number of connections*: weight edges by the number of connections between the groups.
- *Sum of connection weights*: weight edges by the sum of weights of connections between the groups.

*Normalize by geometric mean* divides weights of the edge between two groups by the geometric mean of number of connections these two groups have, that is, by the square root of the product of the groups' degrees.
"""

The first two are clearer now (IMHO). I don't like the latter, but in your version I don't understand what is "the product of degrees of edge pairs.". (This is not to say that my version is much better.)